enterprise/models.py

```diff
@@ -77,6 +77,35 @@ class Meta(object):
     history = HistoricalRecords()
     identity_provider = models.SlugField(null=True, blank=True, default=None)
     site = models.ForeignKey(Site, related_name="enterprise_customers")
+    
+    DATA_CONSENT_OPTIONAL = 'optional'
+    AT_LOGIN = 'at_login'
+    AT_ENROLLMENT = 'at_enrollment'
+    DATA_SHARING_CONSENT_CHOICES = (
+        (DATA_CONSENT_OPTIONAL, 'Optional'),
```

Comment:
Whoops, never mind my comments, I didn't notice that this was split into two fields. I'm going to delete these comments and add new ones.